config/optional/views.view.civiremote_funding_application_list.yml

```diff
@@ -1048,57 +1052,10 @@ display:
             default_group: All
             default_group_multiple: {  }
             group_items: {  }
-        application_process_status:
```

Comment:
I wouldn't expect this to be removed...

Reply:
i see, mixup of the removal of the status in combined application list and this view. i will revert the deletion here